.github/workflows/run_archive.yaml

```diff
@@ -19,13 +19,33 @@ jobs:
       matrix:
         config:
           - {os: ubuntu-20.04,   r: 'release'}
-
+        tag:
```

Comment:
we should change the `name` parameter on L15 to ${{ matrix.tag }}

Reply:
Ah yep, makes sense